experiments/chaos/kubernetes/service_failure/test.yml

```diff
@@ -59,33 +57,49 @@
             executable: /bin/bash
           register: pv_name
 
-        - name: Check for presence & value of cas type annotation
+        - name: Check the persistent volume provisioner name
           shell: >
             kubectl get pv {{ pv_name.stdout }} --no-headers
-            -o jsonpath="{.metadata.annotations.openebs\\.io/cas-type}"
+            -o custom-columns=:.metadata.annotations.pv\\.kubernetes\\.io\/provisioned-by
```

Comment:
is this annotation present for all types of volume.

Reply:
@gprasath 
yes, i verified for cstor, jiva and zfspv